dlc-test/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCClientIntegrationTest.scala

```
@@ -470,10 +479,14 @@ class BinaryOutcomeDLCClientIntegrationTest extends BitcoindRpcTest {
       _ <- waitUntilBlock(
         unilateralDLC.timeouts.contractMaturity.toUInt32.toInt)
       _ <- publishTransaction(unilateralOutcome.cet)
-      _ <- publishTransaction(unilateralOutcome.closingTx)
+      _ <- unilateralOutcome.closingTxOpt
+        .map(publishTransaction)
+        .getOrElse(FutureUtil.unit)
```

do we want to fail silently here? is that what `FutureUtil.unit` will do here?